src/python/bot/fuzzers/libFuzzer/constants.py

```diff
@@ -22,6 +22,8 @@
 # libFuzzer flags.
```

Comment:
One day we should make strategies an object that have a flag, strategy_name and libfuzzer argument.

Reply:
I think @mukundv-chrome did something like this in a recent CL.

Reply:
The one that was landed had only names and probabilities, but didn't have flags / arguments.